### PR TITLE
Request support

### PR DESCRIPTION
**Pull request by exbinary**
Minimal, functional `/asks/new` 🐇 .

Missing some questions when compared to the LAMA [request form](https://docs.google.com/forms/d/10XnUMY4wwp4o2v0UmOQdQ6XcogjkSMtzlp-iXI65v6s/edit):
* "Do you wish to be contacted about donations..."
* "Does your request need to remain anonymous?"
* "Number of people in the household"
* "Is a member in your household over the age of 50?"
* Recurrence and Urgency, which we're planning to attach to categories.

Screen Shot 2020-05-12 at 22 40 12
Screen Shot 2020-05-12 at 22 40 44

### CODE REVIEW

File: app/javascript/components/forms/index.js [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
 import LocationFields   from './LocationFields'
 import NameField        from './NameField'
 import ServiceAreaField from './ServiceAreaField'
 import SpacerField      from './SpacerField'
+import SubmitButton     from './SubmitButton'
 
 export {
   AuthTokenInput,
   CategoryFields,
+  CommentsField,
```
Comment (maebeale) [inline app/javascript/components/forms/index.js:16]: was comments not on offer form?

File: app/javascript/pages/Offer.vue [hunk 2]
```diff
@@ -31,7 +31,20 @@
...
+      <p class="title is-4">
+        What are you able to offer?
+      </p>
+      <p>
+      Note: Risks and best practices will be discussed before any work is done.
+      People over 50 and people with vulnerable conditions should be especially cautious in these times,
+      so this option will not be right for everyone.
+      </p>
+      <p class="mt-1 mb-1">
+      Your safety is community safety. Do what is best.
```
Comment (maebeale) [inline app/javascript/pages/Offer.vue:44]: <3

File: app/javascript/components/forms/index.js (hunk 1 above)
Comment (exbinary) [reply app/javascript/components/forms/index.js:16]: It was, i just hadn't extracted it.

BTW, this is currently saving to the `description` field, which it probably shouldn't be.

Comment (maebeale) [reply app/javascript/components/forms/index.js:16]: yeah, `description` is for the metadata accordion for each checkbox

i'm adding in a `SubmissionResponse` model to be able to import prior data that doesn't match our forms, so this comment could later get stored as one of those

**Pull request merged**